WPMedia/fusion-news-theme-blocks
# Tmedia 84 remove unnecessary deps

## Description
Remove one unnecessary top-level dep. Change tag matching `stable` and `canary` tags to any `*`. This will help with npm 7 migration which will result in smaller bundles

## Jira Ticket
- [TMEDIA-84](https://arcpublishing.atlassian.net/secure/RapidBoard.jspa?rapidView=875&modal=detail&selectedIssue=TMEDIA-84&assignee=5e387d6a1b1d910e5dfd7a9b)

## Acceptance Criteria
- Remove unnecessary deps
- Improve bundle size eventually

## Test Steps

1. Checkout this branch `git checkout TMEDIA-84-remove-unnecessary-deps`
2. Run fusion repo with linked blocks `rm -rf node_modules && npm i && npm t`
3. Ensure tests run as before

## Effect Of Changes
### Before
- Targeting wrong tags with peer deps and npm@7 warnings

### After
- Fewer warnings and unnecessary deps 

## Dependencies or Side Effects
- No side effects 

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
